test/unit/app/messaging/fetch-backoffice-appliation.test.js

```diff
@@ -0,0 +1,54 @@
+const fetchBackOfficeApplication = require('../../../../app/messaging/fetch-backoffice-application')
+const { getBackOfficeApplicationResponseMsgType, backOfficeResponseQueue } = require('../../../../app/config')
+const states = require('../../../../app/messaging/states')
+
+jest.mock('../../../../app/lib/send-email')
+const sendMessage = require('../../../../app/messaging/send-message')
+jest.mock('../../../../app/messaging/send-message')
+const applicationRepository = require('../../../../app/repositories/application-repository')
+jest.mock('../../../../app/repositories/application-repository')
+
+describe(('Store backOffice in database'), () => {
```

Comment:
Remove unnecessary braces around description please.
Also, is this description accurate? Should it be more like `fetch applications for back-office`?